src/utils/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod git;
```

Comment:
I think this `git` module is going to be important enough that it's not a "util" and can just be in `src/git.rs`